src/compiler/types.ts

```diff
@@ -2459,6 +2460,14 @@ namespace ts {
         character: number;
     }
 
+    export const enum ScriptKind {

```

Comment:
Hmm - the tests fail when I do this. Not sure if this should be internal as other hosts can implement getScriptKind.